packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/selectedMessageName.st

```diff
@@ -0,0 +1,7 @@
+code browser
+selectedMessageName
+
+	self currentSelection
+	ifNil: [^ nil].
+	
+	^ self currentSelection method selector
```

Comment:
Please adapt `IPTTreeModel >> getSelector: aMessageTally` to your needs and use it here.